wasm/src/utils.rs

```diff
@@ -75,3 +76,23 @@ pub fn get_memory_limit(spec: &Spec) -> Option<i64> {
         .and_then(|x| x.memory().as_ref())
         .and_then(|x| x.limit())
 }
+
+pub(crate) fn get_rootfs(spec: &Spec) -> containerd_shim::Result<String> {
```

Comment:
Better to return `Option`, let the caller decide how to handle `None`

Reply:
done